static/app/components/events/interfaces/spans/header.tsx

```diff
@@ -818,6 +761,7 @@ const MinimapSpanBar = styled('div')`
   margin: 2px 0;
   min-width: 1px;
   border-radius: 1px;
+  box-sizing: border-box;
```

Comment:
Making this explicit here since we rely on the `<MinimapSpanBar />` component on having a specific height. This is important for when we calculate how many spans can fit into the minimap's container.

Margins will overlap, so the real height for `<MinimapSpanBar />` will be 4 pixels tall.